db/indexes_test.go

```diff
@@ -162,6 +162,9 @@ func TestPostUpgradeIndexesVersionChange(t *testing.T) {
 	if base.TestsDisableGSI() {
 		t.Skip("This test only works with Couchbase Server and UseViews=false")
 	}
+	if !base.TestUseXattrs() {
```

Comment:
What's changed to make this test require xattrs?

Reply:
It relies on the non xattr indexes to be the ones that are "obsolete". I could have change the code to register the opposite of the tests (i.e. create xattr indexes when `SG_TEST_USE_XATTRS=false` and have them be the opposite).

This doesn't seem to mimic a real situation and since xattrs are usually true, it was just to simplify the test reading. I can set this variable dynamically like we do above.